packages/create-invoice-form/src/lib/create-invoice-form.svelte

```diff
@@ -183,19 +184,41 @@
     if (requestNetwork) {
       try {
         addToStatus(APP_STATUS.PERSISTING_TO_IPFS);
-        const request = await requestNetwork.createRequest({
-          requestInfo: requestCreateParameters.requestInfo,
-          paymentNetwork: requestCreateParameters.paymentNetwork,
-          contentData: requestCreateParameters.contentData,
-          signer: requestCreateParameters.signer,
-        });
+        let request;
+        if(formData.isEncrypted) {
+          const payeeEncryptionPublicKey = {
+            key: requestCreateParameters.requestInfo.payee?.value!,
+            method: EncryptionTypes.METHOD.KMS,
+          };
+          const payerEncryptionPublicKey = {
+            key: requestCreateParameters.requestInfo.payer?.value!,
+            method: EncryptionTypes.METHOD.KMS,
+          };
```

Comment:
Consider renaming variables because technically we're passing the Address, not the Public Keys

```suggestion
          const payeeEncryptionParams = {
            key: requestCreateParameters.requestInfo.payee?.value!,
            method: EncryptionTypes.METHOD.KMS,
          };
          const payerEncryptionParams = {
            key: requestCreateParameters.requestInfo.payer?.value!,
            method: EncryptionTypes.METHOD.KMS,
          };
```